.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,6 +27,7 @@
 `{A detailed list of any necessary changes}`
 
 > * `{Please delete this quoted section when opening your pull request}`
-> * You must follow the template instructions. Failure to do so will result in your issue being closed.
-> * Please respect that Pi-hole is developed by volunteers, who can only reply in their spare time.
-> * Detail helps us understand an issue quicker, but please ensure it's relevant.
+
+* You must follow the template instructions. Failure to do so will result in your issue being closed.
+* Please respect that Pi-hole is developed by volunteers, who can only reply in their spare time.
+* Detail helps us understand an issue quicker, but please ensure it's relevant.
```

Comment:
This section looks a bit odd since the "please delete this" line is still in the previous formatting.

Reply:
The concept was that they delete the single line that is in codeblock and leave the rest of the comments in the bottom. I'm going to change most of the bottom anyways since this is the PR template and it talks about Issues being closed.